contracts/token/ERC20/extensions/ERC20FlashMint.sol

```diff
@@ -38,6 +42,10 @@ abstract contract ERC20FlashMint is ERC20, IERC3156FlashLender {
      * @dev Returns the maximum amount of tokens available for loan.
      * @param token The address of the token that is requested.
      * @return The amount of token that can be loaned.
+     *
+     * NOTE: This function does not consider any form of supply cap, so in case
```

Comment:
I'm not sure this is correct. This function uses `_mint` to create the tokens. `_mint` will call `update` with "from = 0". If present, the ERC20Capped module will check that the cap is not exceeded.